src/api/Jalasoft.TeamUp.Resumes.API/Controllers/PatchPostulation.cs

```diff
@@ -0,0 +1,67 @@
+namespace Jalasoft.TeamUp.Resumes.API.Controllers
+{
+    using System;
+    using System.IO;
+    using System.Net;
+    using System.Threading.Tasks;
+    using FluentValidation;
+    using Jalasoft.TeamUp.Resumes.Core.Interfaces;
+    using Jalasoft.TeamUp.Resumes.Models;
+    using Jalasoft.TeamUp.Resumes.ResumesException;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.JsonPatch;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.Http;
+    using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+    using Microsoft.OpenApi.Models;
+    using Newtonsoft.Json;
+
+    public class PatchPostulation
+    {
+        private readonly IPostulationsService postulationsService;
+
+        public PatchPostulation(IPostulationsService postulationsService)
+        {
+            this.postulationsService = postulationsService;
+        }
+
+        [FunctionName("PatchPostulation")]
+        [OpenApiOperation(operationId: "PatchPostulation", tags: new[] { "Postulations" })]
+        [OpenApiParameter(name: "id", In = ParameterLocation.Path, Required = true, Type = typeof(int), Description = "The postulation identifier.")]
+        [OpenApiRequestBody("application/json", typeof(JsonPatchDocument<Postulation>), Description = "JSON request body containing Postulation")]
+        [OpenApiResponseWithBody(statusCode: HttpStatusCode.OK, contentType: "application/json", bodyType: typeof(Postulation), Description = "Successful response")]
```

Comment:
is this the only probable response this endpoint will return? or are we missing document others? 😉